src/v2/view-builder/views/ov/ChallengeOktaVerifyResendPushView.js

```diff
@@ -18,41 +18,25 @@ const Body = BaseForm.extend(Object.assign(
-      const messagesObjs = this.options.appState.get('messages');
-      if (messagesObjs && Array.isArray(messagesObjs.value)) {
-        this.add('<div class="ion-messages-container"></div>', '.o-form-error-container');
-
-        messagesObjs.value.forEach(messagesObj => {
-          const msg = messagesObj.message;
-          if (messagesObj?.class === 'ERROR') {
-            const options = {
-              content: msg,
-              type: 'error',
-            };
-            if (this.options.appState.containsMessageWithI18nKey(OV_NMC_FORCE_UPGRAGE_SERVER_KEY)) {
-              // add a title for OV force upgrade
-              options.title = loc('oie.numberchallenge.force.upgrade.title', 'login');
-            } else if (this.options.appState.containsMessageWithI18nKey(OV_UV_ENABLE_BIOMETRIC_SERVER_KEY)) {
-              // add a title for OV enable biometrics message during verification
-              options.content = null;
-              options.className = 'okta-verify-uv-callout-content';
-              options.title = loc('oie.authenticator.app.method.push.verify.enable.biometrics.title', 'login');
-              options.subtitle = loc('oie.authenticator.app.method.push.verify.enable.biometrics.description', 'login');
-              options.bullets = [
-                loc('oie.authenticator.app.method.push.verify.enable.biometrics.point1', 'login'),
-                loc('oie.authenticator.app.method.push.verify.enable.biometrics.point2', 'login'),
-                loc('oie.authenticator.app.method.push.verify.enable.biometrics.point3', 'login')
-              ];
-            }
-            this.add(createCallout(options), '.o-form-error-container');
-          } else {
-            this.add(`<p>${msg}</p>`, '.ion-messages-container');
-          }
-        });
-
+      let options = {};
+      if (this.options.appState.containsMessageWithI18nKey(OV_NMC_FORCE_UPGRAGE_SERVER_KEY)) {
+        // add a title for OV force upgrade
+        options.title = loc('oie.numberchallenge.force.upgrade.title', 'login');
+      } else if (this.options.appState.containsMessageWithI18nKey(OV_UV_ENABLE_BIOMETRIC_SERVER_KEY)) {
+        // add a title for OV enable biometrics message during verification
+        options.content = null;
+        options.className = 'okta-verify-uv-callout-content';
+        options.title = loc('oie.authenticator.app.method.push.verify.enable.biometrics.title', 'login');
+        options.subtitle = loc('oie.authenticator.app.method.push.verify.enable.biometrics.description', 'login');
+        options.type = 'error';
+        options.bullets = [
+          loc('oie.authenticator.app.method.push.verify.enable.biometrics.point1', 'login'),
+          loc('oie.authenticator.app.method.push.verify.enable.biometrics.point2', 'login'),
+          loc('oie.authenticator.app.method.push.verify.enable.biometrics.point3', 'login')
+        ];
+        options = createCallout(options);
```

Comment:
Why do you need to create `createCallout` instead of just pass `options`?

Reply:
Gives ability to pass any view not just callout.. baseForm callout has only bare minimum properties.. content, type and title

Reply:
I meant 
instead of doing this 
```
   if(options instanceof View) {
             // add custom callout or any view
             this.add(options, errContainer);
           } else {
             // add error callout with custom options 
             options = Object.assign(_.pick(obj, 'message', 'class'), options);
             this.add(createCallout(...))
           }
```

can we do

```
const callout = createCallout(createCallout(_.defaults(options), {title:'', type:option.class})
this.add(callout)
```

The reason I'm asking is because looks like we didn't have use case where child class want to pass any view.
If you're thinking `in future, maybe child class want to pass any view`, I'd suggest to leave it for future given it's small refactoring.

Reply:
@haishengwu-okta Yes, I wanted to provide an ability to render any view as terminal state.. We currently have couple of flows that implements their own `templates` to render terminal views that doesn't have any callouts.. 

for eg take a look at this.. 
https://github.com/okta/okta-signin-widget/pull/2235#pullrequestreview-792043622